.github/workflows/ci.yml

```diff
@@ -75,5 +75,25 @@ jobs:
         file: ./coverage.xml
         yml: ./codecov.yml
 # Disabled this line because Codecov has been extra flaky lately, and having to re-run the entire
-# test suite until every coverage upload step succeeds is more of a hassle than it's worth. 
+# test suite until every coverage upload step succeeds is more of a hassle than it's worth.
 #        fail_ci_if_error: true
+  upload:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'schedule'
+    needs: tests
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Build Dist
+      run: |
+        python3 -m pip install wheel
+        python3 setup.py --dev_release sdist bdist_wheel
+    - name: Upload to PyPI
+      uses: pypa/gh-action-pypi-publish@v1
```

Comment:
```suggestion
      uses: pypa/gh-action-pypi-publish@v1.3.0
```

Reply:
You sure about this? Standard practice for other actions seems to be to only pin the major version.

Also, though 1.3.0 is the latest tag, 1.2.2 is the latest version available in the marketplace.

Reply:
> only pin the major version

Oh, I didn't realize that. I guess I just assumed that they did a more simple lookup based on exact tag/branch

Ignore this then :)